content/en/community/_index.html

```diff
@@ -122,9 +122,9 @@ <h2>Discussions</h2>
 
     <div id="twitter" class="community-resource">
       <a href="https://twitter.com/kubernetesio">
-        <img src="/images/community/twitter.png" alt="Twitter">
+        <img src="static/images/community/x1.png" alt="X.org">
```

Comment:
```suggestion
        <img src="static/images/community/x1.png" alt="𝕏">
```
https://x.org/ is a different organization.